Azure/sonic-buildimage
# Tuning values for optics on 7060X6_PE 256x200G hwsku

#### Why I did it
There is ongoing testing of this product using optics with 100g lane speeds. This media_settings.json file contains the optimal tunings for all ports on this product when the lane speed is 100g and the medium type is optic.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Added the media_settings.json file in the hwsku folder.

#### How to verify it
You can check APPL_DB or 'bcmcmd dsh -c "phy diag <logical_port> dsc config"' to see the tuning values applied on each lane.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)
202311




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
